core/src/main/java/io/questdb/cairo/vm/api/MemoryCARW.java

```diff
@@ -227,12 +253,27 @@ default void putStr(long offset, CharSequence value) {
         }
     }
 
+    default void putStrLowerCase(long offset, CharSequence value) {
```

Comment:
Considering that only FastMap needs this method, it might be a better idea to implement it in `MemoryCARWSpillable`. Same with `putBin32` and other methods.